src/ensembl/src/content/app/entity-viewer/shared/helpers/entity-helpers.ts

```diff
@@ -16,16 +16,17 @@
 
 import { Pick2, Pick3 } from 'ts-multipick';
 
-import { Slice } from 'src/shared/types/thoas/slice';
-import { PhasedExon, Exon } from 'src/shared/types/thoas/exon';
-import { Product, ProductType } from 'src/shared/types/thoas/product';
-import { ExternalReference } from 'src/shared/types/thoas/externalReference';
-
 import {
   SWISSPROT_SOURCE,
   SPTREMBL_SOURCE
 } from 'src/content/app/entity-viewer/gene-view/components/proteins-list/protein-list-constants';
 
+import { Slice } from 'src/shared/types/thoas/slice';
+import { PhasedExon, Exon } from 'src/shared/types/thoas/exon';
+import { Product, ProductType } from 'src/shared/types/thoas/product';
+import { ExternalReference } from 'src/shared/types/thoas/externalReference';
+import { TranscriptMetadata } from 'ensemblRoot/src/shared/types/thoas/metadata';
```

Comment:
Notice that when you rely on VSCode to infer your import, it starts the path with `ensemblRoot`. I suppose it pulls it out of `tsconfig.json`. Don't know what to do about this.

Reply:
Yeah this is about annoying. The auto import function is quite handy after all, so don't want to disable it either.

Reply:
✅